packages/fury-adapter-oas3-parser/STATUS.md

```diff
@@ -24,6 +24,38 @@ Key:
 | tags | [✕](https://github.com/apiaryio/api-elements.js/issues/75) |
 | externalDocs | [✕](https://github.com/apiaryio/api-elements.js/issues/82) |
 
+## Info Object
+
+| title | ✓ |
+| description | ✓ |
+| termsOfService | ✓ |
+| contact | ✓ |
+| license | ✓ |
+| version | ✓ |
+
+## Contact Object
+
+| name | ✓ |
+| url | ✓ |
+| email | ✓ |
+
+## Licence Object
```

Comment:
```suggestion
## License Object
```

License is spent differently in the spec: https://spec.openapis.org/oas/v3.0.2#license-object

Reply:
sure, changed